applications/klighd-vscode/package.json

```diff
@@ -178,6 +178,7 @@
     },
     "dependencies": {
         "@kieler/klighd-core": "^0.4.2",
+        "@kieler/klighd-interactive": "^0.4.2",
```

Comment:
this is a transient dependency, so we should not be required to repeat this dependency here